File: JavaScript/app/models/lizard.js
var Shape = require('./shape');

var Lizard = Object.create(Shape);

Lizard.setup = function (ShapeType) {
	this.init({
		shapeType: ShapeType.lizard,
		winsTo: [ShapeType.paper, ShapeType.spock]
	});
}

module.exports = Lizard;

File: NET/Kata.RockPaperScissor/Lizard.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Kata.RockPaperScissor
{
    public class Lizard : Shape
    {
        public Lizard()
        {
            ShapeType = ShapeType.Lizard;
            Name = "Lizard";
            winsTo = new List<ShapeType>() { ShapeType.Paper, ShapeType.Spock };
        }
    }
}
